src/Translator/doc/index.rst

```diff
@@ -138,6 +138,12 @@ file. 2 of the new items are::
         'path' => 'var/translations/configuration.js',
     ],
 
+.. caution::
+
+    If you change the ``dump_directory`` in your configuration file, you will need to
```

Comment:
And add an: if you use AssetMapper.

Reply:
It is already in the "Using with AssetMapper" so i'd say we don't need here